scripts/docker-build-and-push.sh

```diff
@@ -21,8 +21,9 @@ Options:
         Build and push only this Dockerfile. Otherwise all found will be used.
 
   -r, --registry
-        The Docker registry to push to, without the trailing slash.
+        List of one or more Docker registries to push to, without the trailing slash.
         Remember to change the corresponding value in conf/resinhup.conf as well.
+        Default: "$REGISTRY".
```

Comment:
This is not actually working as you would expect. If you runt the script with `--registry "foo" --help`, it would show a wrong default value.